### PR TITLE
Disable `Style/EachWithObject`

### PR DESCRIPTION
**Pull request by Reprazent**
We're using "Style/CollectionMethods" to specify our preference:

Style/CollectionMethods:
  PreferredMethods:
    reduce: 'inject'
    each_with_object: "inject"

### CODE REVIEW

**Comment by To1ne** [comment]
But now it does not complain about `each_with_object`. Actually about none of the methods in `Style/CollectionMethods`.`PreferredMethods`

**Comment by To1ne** [comment]
:+1:
(Adding tests to this project wouldn't be bad.)

**Comment by To1ne** [comment]
Deployed to Marleen.

**Comment by Reprazent** [comment]
Didn't marleen do the automatic thingy?

**Comment by To1ne** [comment]
How?

**Comment by Reprazent** [comment]
I hacked that using `inherit_from` explained here: https://github.com/10to1/standards/commit/338c664491e0bf27b567adcdb4af737b7aef03a0#diff-04c6e90faac2675aa89e2176d2eec7d8R54



-- 
Bob Van Landuyt
+32 473 43 05 97

**Comment by Reprazent** [comment]
I put back the inheritance.

More info in the rubocop docs: http://rubocop.readthedocs.io/en/latest/configuration/#inheritance

-- 
Bob Van Landuyt
+32 473 43 05 97

**Comment by To1ne** [comment]
Ah sorry, missed that!

**Comment by Reprazent** [comment]
To be honest, I needed to look it up as well ;-).
I knew i'd have written it down somewhere ;-)

-- 
Bob Van Landuyt
+32 473 43 05 97

File: guides/ruby.yml [hunk 1]
```diff
@@ -105,7 +105,7 @@ Style/Documentation:
   Enabled: false
 
 Style/EachWithObject:
-  Enabled: false
+  Enabled: true
```
Comment (Reprazent) [inline guides/ruby.yml]: This commit is broken @To1ne

**Pull request merged**